src/main/java/Duke.java

```diff
@@ -1,10 +1,378 @@
+                }
+
+                System.out.println(line);
+            }
+
+            else if (text.contains("todo")) {
+                if(text.equals("todo") || text.equals("todo ")){
+                    System.out.println(line);
+                    System.out.println("☹ OOPS!!! The description of a todo cannot be empty.");
+                    System.out.println(line);
+                } else{
+                    System.out.println(line);
+                    System.out.println("Got it. I've added this task");
+                    text = text.substring(5, text.length());
+                    System.out.println("  [T][ ] " + text);
+                    System.out.println("Now you have " + count + " tasks in the list.");
+                    System.out.println(line);
+                    tasks[count] = text;
+                    type[count] = "[T]";
+                    marks[count] = "[ ]";
+                    count++;
+                }
+                
+            }
+
+            else if (text.contains("deadline")) {
+                System.out.println(line);
+                System.out.println("Got it. I've added this task");
+
+                // Modify text according to deadline
+                text = text.substring(9, text.length());
+                int indexOfSlash = text.indexOf("/",9);
+                String day = text.substring(indexOfSlash+4, text.length());
+                String task = text.substring(0, indexOfSlash);
+                text = task + "(by: " + day + ")";
+
+                System.out.println("  [D][ ] " + text);
+                System.out.println("Now you have " + count + " tasks in the list.");
+                System.out.println(line);
+                tasks[count] = text;
+                type[count] = "[D]";
+                marks[count] = "[ ]";
+                count++;
+            }
+            else if (text.contains("event")) {
+                System.out.println(line);
+                System.out.println("Got it. I've added this task");
+
+                // Modify text according to deadline
+                text = text.substring(6, text.length());
+                int indexOfSlash = text.indexOf("/",9);
+                String day = text.substring(indexOfSlash+4, text.length());
+                String task = text.substring(0, indexOfSlash);
+                text = task + "(at: " + day + ")";
+
+                System.out.println("  [E][ ] " + text);
+                System.out.println("Now you have " + count + " tasks in the list.");
+                System.out.println(line);
+                tasks[count] = text;
+                type[count] = "[E]";
+                marks[count] = "[ ]";
+                count++;
+            }
+            else{
+                System.out.println(line);
+                System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+                System.out.println(line);
+            }
+        }
+
+
+
```

Comment:
Avoid long methods. As mentioned on the module website: 

> Be wary when a method is longer than the computer screen, and take corrective action when it goes beyond 30 LOC (lines of code). The bigger the haystack, the harder it is to find a needle.

Please consider shorterning the method by extracting logic into sub-methods (SLAP more).